### DISCUSSION TITLE
A32NX/MCDU/AOC-MENUE/FREE TEXT

### DISCUSSION THREAD

**Discussion by BEHAD** [flybywiresim/aircraft#5058] [Q&A]
Hello,
in the A32NX it is now possible via MCDU and AOC-Menu to send a "free text" to another live player, provided I know his flight number.
If I select an airfield in live mode and zoom in, I can see all "live players" (Green Airplane Icons).
Run the Mouse over it, a name/number is also displayed. But I think this is the username!
Is there a possibility to determine the "flight number" of the aircrafts to send him a small message. Wouldn't it be nice to write -for example-"Hello, I wish you a good flight. I’m on your left".

Does somebody has any idea????

Kind Regards B. H.

Hallo (german),
im A32NX ist es per MCDU im AOC-Menü ja jetzt möglich, einen "freien Text" an einen anderen Live-Spieler zu senden, sofern ich dessen Flugnummer kenne.
Wenn ich im live Modus einen Flugplatz auswähle und zoome, kann ich alle "Live-Mitspieler" sehen (grüne Flugzeug-Icons).
Wenn ich mit der Maus darauf zeige, wird mir auch ein Name/Nummer angezeigt. Dies scheint aber der Benutzername zu sein!
Gibt es eine Möglichkeit vom jeweiligen Flugzeug die "Flugnummer" zu ermitteln, um Ihm eine kleine Nachricht zu senden.
Wäre doch ganz nett z.B. mal zu schreiben: "Hallo, wünsche guten Flug, stehe links neben Dir"!

Hat jemand eine Idee????

LG B. H.

**Comment by ExampleWasTaken** [comment]
What do you mean with "Live Mode"? Real-time traffic in the simulator?

The A32NX does not feed other online A32NX into the world map. You can see all other A32NX' that have TELEX enabled [here](https://flybywiresim.com/map).

**Reply by BEHAD** [reply]
Hallo, danke für die Antwort. 
Meine Frage noch einmal etwas detaillierter:
Nach Klick auf Weltkarte und Auswahl eines Abflugflugplatz z.B. EDDF
Unter "Flugbedingungen" Mehrspieler=Alle Spieler / Flugverkehr=Live Verkehr / Wetter Zeit= Live
Nach vollem Zoomen sehe ich an den Gates alle geparkten Flugzeuge als grüne Icons
Ich setzte meinen Abflug zwischen zwei geparkte Flugzeuge fest.
Nach Klick auf "Losfliegen" und "Bereit zum fliegen" bin ich im Cockpit
Wenn ich nun in der "Außenansicht" schwenke, sehe ich die neben mir geparkten Flugzeuge,
zwischen denen ich mein Parking-Gate ausgewählt habe.
Das sind dann auch meist Maschinen mit dem Aufdruck "flybywire" (also auch A32NX)
Wie kann ich mit denen -mit free text- in Kontakt treten, wozu ich aber deren Flugnummer bräuchte.
Kann man diese irgendwo / irgendwie herausbekommen/erfahren?
Oder geht das nur im "Mehrspielermodus" Oder ist das Ganze von mir eine blöde Idee? 
Danke im Voraus für eine Erklärung
MfG B.H.

**Reply by ExampleWasTaken** [reply]
> 
> 
> Hallo, danke für die Antwort.
> Meine Frage noch einmal etwas detaillierter:
> Nach Klick auf Weltkarte und Auswahl eines Abflugflugplatz z.B. EDDF
> Unter "Flugbedingungen" Mehrspieler=Alle Spieler / Flugverkehr=Live Verkehr / Wetter Zeit= Live
> Nach vollem Zoomen sehe ich an den Gates alle geparkten Flugzeuge als grüne Icons
> Ich setzte meinen Abflug zwischen zwei geparkte Flugzeuge fest.
> Nach Klick auf "Losfliegen" und "Bereit zum fliegen" bin ich im Cockpit
> Wenn ich nun in der "Außenansicht" schwenke, sehe ich die neben mir geparkten Flugzeuge,
> zwischen denen ich mein Parking-Gate ausgewählt habe.
> Das sind dann auch meist Maschinen mit dem Aufdruck "flybywire" (also auch A32NX)
> Wie kann ich mit denen -mit free text- in Kontakt treten, wozu ich aber deren Flugnummer bräuchte.
> Kann man diese irgendwo / irgendwie herausbekommen/erfahren?
> Oder geht das nur im "Mehrspielermodus" Oder ist das Ganze von mir eine blöde Idee?
> Danke im Voraus für eine Erklärung
> MfG B.H.

According to Google you are talking about the world map of the simulator. On there you will see all other players that have multiplayer enabled in the simulator. It doesn't matter what aircraft they fly. If you load into the simulator, the models will not necessarily match. Same applies to liveries. 
You can only send free text messages to other A32NX, not every aircraft you see on the world map when selecting an aircraft. You can see all A32NX that have TELEX enabled, on the site I linked to in my previous reply.

**Comment by BEHAD** [comment]
Hallo, danke nochmals für Ihre Antwort.
Jetzt hab ich es endlich verstanden und begriffen!!!!
Hab mir in Ihren Link (flybywiresim.com) die Karte angesehen und auch die aktuellen A32NX  ****mit Flugnummern**** gesehen.
Vielen Dank für die Hilfe. Mfg B.H.  LG
